crates/nu-command/src/conversions/into/string.rs

```diff
@@ -210,6 +218,13 @@ pub fn action(
         Value::Int { val, .. } => {
             let res = if group_digits {
                 format_int(*val) // int.to_formatted_string(*locale)
+            } else if let Some(dig) = digits {
+                let mut val_with_trailing_zeroes = val.to_string();
+                val_with_trailing_zeroes.push('.');
```

Comment:
If user input `5 | into string -d 0`, the result will be `5.`, I don't think it's good

Reply:
Good catch, @WindSoilder !